src/ClassicalOrthogonalPolynomials.jl

```diff
@@ -318,6 +319,51 @@ end
 
 golubwelsch(V::SubQuasiArray) = golubwelsch(parent(V), maximum(parentindices(V)[2]))
 
+function gaussradau(P::Monic{T}, n::Integer, endpt) where {T}
+    ## See Thm 3.2 in Gautschi (2004)'s book 
+    # n is number of interior points 
+    J = jacobimatrix(P.P, n + 1)
+    α, β = diagonaldata(J), supdiagonaldata(J)
+    endpt = T(endpt)
+    p0 = P[endpt, n] # πₙ₋₁
+    p1 = P[endpt, n+1] # πₙ. Could be faster by computing p1 from p0 and πₙ₋₂, but the cost is tiny relative to eigen()
```

Comment:
```suggestion
p0, p1 = P[endpt,n:n+1]
```
If we overload `recurrencecoefficients` below this will automatically be fast.